src/python/WMCore/WMRuntime/SandboxCreator.py

```diff
@@ -162,8 +165,12 @@ def makeSandbox(self, buildItHere, workload):
 
             # Add a dummy module for zipimport testing
             (handle, dummyModulePath) = tempfile.mkstemp()
-            os.write(handle, "#!/usr/bin/env python\n")
-            os.write(handle, "print('ZIPIMPORTTESTOK')\n")
+            if PY2:
+                os.write(handle, "#!/usr/bin/env python\n")
+                os.write(handle, "print('ZIPIMPORTTESTOK')\n")
+            else:
+                os.write(handle, b"#!/usr/bin/env python\n")
+                os.write(handle, b"print('ZIPIMPORTTESTOK')\n")
```

Comment:
I do not remember where an when, but i have already spent time on `tempfile.mkstemp` and it's a pity that they force the input to be [bytes in py3](https://docs.python.org/3/library/tempfile.html#tempfile.mkstemp), too.

In any case, if you have a fetish for brevity, and since this is one of those occasions where we want to force the output to be bytes both in py2 and py3, we can opt for

```suggestion
            os.write(handle, decodeUnicodeToBytes("#!/usr/bin/env python\n"))
            os.write(handle, decodeUnicodeToBytes("print('ZIPIMPORTTESTOK')\n"))
```

But i understand that this is not necessary. as always the final about code style is up to you!

Reply:
Applying, but the correct method name `encodeUnicodeToBytes` :-P

Reply:
ahaha, you are correct, it did't sound right but i was too lazy to check :grimacing: